packages/documentation/pages/usage/components/form-fields.vue

```diff
@@ -840,13 +840,18 @@ export default defineComponent({
 			) {
 				const options =
 					component === 'KtFieldSingleSelectRemote'
-						? singleOrMultiSelectOptions.filter((option) =>
-								option.label
-									.toLowerCase()
-									.includes(
-										(remoteSingleSelectQuery.value ?? '').toLowerCase(),
-									),
-						  )
+						? singleOrMultiSelectOptions
+								.filter((option) =>
+									option.label
+										.toLowerCase()
+										.includes(
+											(remoteSingleSelectQuery.value ?? '').toLowerCase(),
+										),
+								)
+								.map((option) => ({
+									...option,
+									dataTest: `dataTest ${String(option.value)}`,
```

Comment:
AFAIK those data-test attributes should be unique, right? So It would make sense to have a marker per field as well so in case there are two fields on the same page, to make them distinguishable. 

In other words I would propose we also use the `dataTest` attribute that you added as a prop for remote single selects here like so `dataTest: `dataTest ${ogFieldDataTest} ${String(option.value)}`

Reply:
what would `ogFieldDataTest` be

Reply:
oh wait nvm i actually forgot to add it to the additional props to enable user-input here

Reply:
if you want to prefix with the field dataTest, I'd pass the formId for all of them  - on it

Reply:
`ogFieldDataTest` is sadly not exposed on the API

Reply:
and actually doesn't make sense to be - since it's internally generated and serves the purpose of identifying an HTML element - which is usually queried post rendering anyways